air-quality-backend/tests/api/mappers/forecast_mapper_test.py

```diff
@@ -1,38 +1,22 @@
 from datetime import datetime, timezone
 
 from air_quality.api.mappers.forecast_mapper import map_forecast
-from tests.api.mappers.create_mock_forecast_document import (
-    create_mock_forecast_document,
-)
+from tests.util.mock_forecast_data import create_mock_forecast_document
 
 
-def test_map_forecast_database_data_to_api_output_data():
-    result = map_forecast(
-        [create_mock_forecast_document({}), create_mock_forecast_document({})]
-    )
+def test__map_forecast_database_data_to_api_output_data():
+    result = map_forecast([create_mock_forecast_document({})])
     assert result == [
```

Comment:
Are we losing coverage of the case where multiple items are mapped if we do this?